docs/development_guide.md

```diff
@@ -0,0 +1,77 @@
+# Developing Decidim
+
+## Create a development_app
+
+In order to start developing you will need what is called a `development_app`. This is nearly the same as a new Decidim app (that you can create with `decidim app_name`) but with a Gemfile pre-configured for local development and some other small config modifications.
+You need it in order to have a Rails application configured to lookup Decidim modules from your filesystem. This way changes in your modules will be directly observed by this `development_app`.
+
+You can create a `development_app` from inside the project's root folder with the command:
+
+```console
+git clone https://github.com/decidim/decidim.git
+cd decidim
+bundle install
+bundle exec rake development_app
+cd development_app
+```
+
+A development_app/ entry appears in the .gitignore file, so you don't have to worry about commiting the development app by mistake.
+
+Once created, you will only have to
+
+- configure its `config/database.yml`
+- `bundle install`
+- `bin/rails db:migrate db:seed`
```

Comment:
All of the previous three lines are applied by default by the generator... In OSX I think it will just work, on Linux you probably need to tweak PostgreSQL config and depending on the configuration chosen, set the `DATABASE_USERNAME` and `DATABASE_PASSWORD` environment variables.